index.html

```diff
@@ -150,7 +150,31 @@ <h6>languaages</h6>
                 </li>
                  </ul>
             </section>
+            <section class="contact">
+              <h1 class="me">Contact me</h1>
+              <p>If you have an application you
+                are interested in developing, a
+                feature that you need built or aproject that needs coding. 
+                
+                I’d love to help with it!</p>
+                <form class="contact_form">
+                     <label for=""></label>
+                    <input type="text" placeholder="Lorem Ipsum" value="" class="form1">
+
+                    <input type="email" placeholder="loremipsum@gmail.com" class="form2">
```

Comment:
- Nice work so far 👍🏿 
- [ ]  I can see that the "required" attribute is indeed not present in the provided code. The inclusion of the "required" attribute ensures that the user must fill out the input field before submitting the form, meeting the requirement for a required field. 😄